File: HYBAP_DYN_GUI/packages/VirtualMotion.VMLNet.ManagerKey.4.0.0.2/content/ManagerKeyInfo.cs
using VM.Managed.Licnese;
namespace VM.Managed.License
{
	class LicenseManagerInitializer
	{
		static string Key{
			get
			{
				return ""
				;
			}
		}
		public static void Initialize()
		{
			CompanyLicenseAttribute.Initialize(
				Key
			);
		}
		public static void Initialize(string strArgument)
		{
			CompanyLicenseAttribute.Initialize(
				Key, strArgument
			);
		}
	}
}

File: HYBAP_DYN_GUI/packages/VirtualMotion.VMLNet.ManagerKey.4.0.0.2/content/native/ManagerKeyInfo.cpp
#include "stdafx.h"

#include "ManagerKeyInfo.h"

namespace VM
{
	namespace Managed
	{
		namespace License
		{
			void InitializeLicense()
			{
				CompanyLicenseAttribute::Initialize(
					VMLICENSING_LICENSEMANAGER_KEY);
			}
			void InitializeLicense(System::String^ strOptions)
			{
				CompanyLicenseAttribute::Initialize(
					VMLICENSING_LICENSEMANAGER_KEY, strOptions);
			}
		}
	}
}
